docs/_docs/administrators-guide/config/node-config.adoc

```diff
@@ -100,10 +100,12 @@ See the link:developers-guide/clients/overview[Clients] section for information
 ----
 {
   "ignite" : {
-    "criticalWorkers" : {
-      "livenessCheckInterval" : 200,
-      "maxAllowedLag" : 500,
-      "nettyThreadsHeartbeatInterval" : 100
+    "system" : {
```

Comment:
I'm pretty sure this section should be merged with `https://github.com/apache/ignite-3/blob/main/docs/_docs/administrators-guide/config/node-config.adoc#system-configuration`